planning/src/chronicles/concrete.rs

```diff
@@ -224,14 +224,32 @@ impl Substitute for Vec<Atom> {
     }
 }
 
-#[derive(Clone, Debug)]
+#[derive(Clone)]
 pub struct Effect {
     pub transition_start: Time,
     pub persistence_start: Time,
     pub state_var: Sv,
     pub value: Atom,
 }
 
+impl Debug for Effect {
```

Comment:
"[start, end] sv := value"

Reply:
The current debugging is currently formatting the SymIDs and TypeIDs. 

```bash
start: FAtom { num: IAtom { var: IVar(VarRef(41)), shift: 0 }, denom: 10 }      
end: FAtom { num: IAtom { var: IVar(VarRef(41)), shift: 1 }, denom: 10 }
state_var: 
        SCst: TypedSym { sym: SymId(25), tpe: TypeId(5) }
        SVar: SVar { var: VarRef(48), tpe: TypeId(9) }
value: Bool(true)
```

Reply:
A sensible output for your example would be the following:
```
start: ?f41    
end: ?f41 + 0.1
state_var: sym_25(?s48)
        SCst: TypedSym { sym: SymId(25), tpe: TypeId(5) } -> sym25
        SVar: SVar { var: VarRef(48), tpe: TypeId(9) }           -> ?s48
value: true
```
where variables a have a prefix `?` (indicating that it is a variable) and a prefix `f`, `s`, `i` indicating their type followed by the variable ID.
When putting those together in an effect, we should obtain the following:

```
[?f41, ?f41 + 0.1] sym25(?s48) <- true
```

Reply:
I will reformat the output as proposed. Will there be an option to directly map the names from `symbol_table`?

Reply:
That would be great, but cannot be done in the `Debug` trait because it needs some context (the symbol table) while `Debug` can only rely on the current value.
This context will not always be available when Debug output is needed.

So using the context info is out of scope for this PR (The `extension/format.rs` file does something like this with the context information but is currently limited to Atom I believe).

Reply:
Understood. I will push the changes soon.